### PR TITLE
Add appveyor.yml for documentation CI

### PR DESCRIPTION
**Pull request by qinezh**
@markcowl @cormacpayne

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@qinezh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@qinezh Can you please follow the directiosn here:  http://aka.ms/azuregithub for linking your ms account with your github ID?  Right now you're being identified as external by the CLA bot, so we can't look at the PR without this, or a signed CLA.

**Comment by qinezh** [comment]
I have already linked my GitHub and corporate identities. As you can see, I joined the Azure Organization too.

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+clone_folder: c:\projects\azure-sdk-for-net
+
+environment:
+  access_token:
+    secure: JWSy9tPo2eoTYpPQplvokf4IiVk6onNBbWwAs6nNWWyQ2JaO+MRAg/QzbJnTjpUm
```
Comment (markcowl) [inline appveyor.yml]: @qinezh Is this an actual access token?

**Comment by markcowl** [comment]
@qinezh Does this need to be in the targeted branch?  We ship from both the master and AutoRest branches, so does it make sense to have one of these for each branch?

**Comment by qinezh** [comment]
I think it's enough to merge it into master branch, as we only build documentation for master branch currently.

**Comment by sphibbs** [comment]
master branch only builds Hyak specs. If you want to document the latest you would want to have this in the autorest master branch where swagger specs are built

**Comment by qinezh** [comment]
As we need to add appveyor.yml into AutoRest branch, I create a [new PR here](https://github.com/Azure/azure-sdk-for-net/pull/2294) and close this.

**Pull request closed without merging**